mypyc/irbuild/util.py

```diff
@@ -13,6 +13,7 @@
     'dataclasses.dataclass',
     'attr.s',
     'attr.attrs',
+    'attr.define',
```

Comment:
Add tests for this

Reply:
I ended up removing the `attr.define` support.  I did some tests and something about the way the mypyc support for attrs and dataclasses works is tripping up attrs auto-detection of annotations.  I'm wary of delaying this PR further because I'm afraid I'll run out of steam, and we can always add `attr.define` support later.